hoot-core/src/main/cpp/hoot/core/io/HootApiDb.cpp

```diff
@@ -514,7 +527,6 @@ void HootApiDb::beginChangeset(const Tags& tags)
   _insertChangeSet->bindValue(":max_lat", _changesetEnvelope.getMaxY());
   _insertChangeSet->bindValue(":min_lon", _changesetEnvelope.getMinX());
   _insertChangeSet->bindValue(":max_lon", _changesetEnvelope.getMaxX());
-  _insertChangeSet->bindValue(":tags", _escapeTags(tags));
```

Comment:
Why remove this part? By including strings directly in SQL it increases the risk of SQL injection.